src/hivpy/hiv_status.py

```diff
@@ -16,29 +17,51 @@
 
 
 class HIVStatusModule:
+
     initial_hiv_newp_threshold = 7  # lower limit for HIV infection at start of epidemic
     initial_hiv_prob = 0.8  # for those with enough partners at start of epidemic
 
     def __init__(self):
-        self.stp_HIV_rate = {SexType.Male: np.zeros(5),
-                             SexType.Female: np.zeros(5)}  # FIXME
-        self.stp_viral_group_rate = {SexType.Male: np.array([np.zeros(7)]*5),
-                                     SexType.Female: np.array([np.zeros(7)]*5)}
-        # FIXME move these to data file
+        # self.stp_HIV_rate = {SexType.Male: np.zeros(5),
+        #                      SexType.Female: np.zeros(5)}  # FIXME
+        # self.stp_viral_group_rate = {SexType.Male: np.array([np.zeros(7)]*5),
+        #                              SexType.Female: np.array([np.zeros(7)]*5)}
```

Comment:
Shouldn't we just delete these since they've been replaced by `ratio_infected_stp` and `ratio_vl_stp` instead of leaving them commented out here?

Reply:
Yes! Good spot I had forgotten about them.